hydpy/core/hydpytools.py

```diff
@@ -211,15 +211,17 @@ class |Parameter| for more information on dealing with time-dependent parameters
     >>> pub.options.parameterstep("1h")
     Period("1d")
 
-    The values of the derived parameters, which need to be calculated before starting a
-    simulation run based on the control parameters and eventually based on some other
-    settings (e.g. the initialisation period), are also ready.  Here we show the value
-    of the derived parameter  |hland_derived.UH|, representing the ordinates of a unit
-    hydrograph (the single value of 1.0 means that the unit hydrograph does not cause
-    any time delay):
-
-    >>> model.parameters.derived.uh
-    uh(1.0)
+    Unlike previous versions of Hydpy, the former parameter |hland_derived.UH| is now
+    part of the rconc submodel and is no longer a derived parameter, but a control
+    parameter that is parameterized directly upon setting, in this case, already in the
+    control file. Here we show the value of the control parameter  |rconc_control.UH|,
+    representing the ordinates of a unit hydrograph (the single value of 1.0 means that
+    the unit hydrograph does not cause any time delay):
+
+    >>> model.rconcmodel.parameters.control.uh
+    uh("triangle", tb=0.36728)
+    >>> model.rconcmodel.parameters.control.uh.value
+    array([1.])
```

Comment:
The intention was to introduce the "derived parameters" (and say they are ready for work). Maybe you can just select another parameter. `Z` could be a good candidate because it's easy to understand (at least, easier than the old and new versions of `UH`).